mytoyota/models/endpoints/trips.py

```diff
@@ -23,6 +36,31 @@ class _SummaryBaseModel(BaseModel):
         alias="fuelConsumption", default=None
     )  # Electric cars might not use fuel. Milliliters.
 
+    def __add__(self, other: _SummaryBaseModel):
+        """
+        Adds together two SummaryBaseModel's. Handles Min/Max/Average fields correctly
+
+        Parameters
+        ----------
+        other: _SummaryBaseModel: to be added
+        """
+        if other is not None:
+            self.length += other.length
```

Comment:
`mytoyota/models/endpoints/trips.py:48:12: E1101: Instance of '_SummaryBaseModel' has no 'length' member (no-member)`

Mhmm, really strange , since `length` is clearly a member of `_SummaryBaseModel`.

Reply:
I've switched to ruff for linking, which I've wanted to do for a long time anyway. As a result, this comment has become obsolete.